addons/Socket/source/LocalNameServers.c

```diff
@@ -157,7 +157,8 @@ void LocalNameServers_findIps(LocalNameServers *self)
 
 void LocalNameServers_findIps(LocalNameServers *self)
 {
-	char *path = tmpnam(NULL);
+        char *path = NULL;
```

Comment:
Funny, the whitespace doesn't show up incorrectly on my computer. Probably some issue with my vim configuration.